### PR TITLE
Pull in upstream changes

### PR DESCRIPTION
**Pull request by SteelWagstaff**
Pull in upstream changes since last merge. Had to resolve one conflict in Vagrantfile. See comment.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -16,7 +16,11 @@ ensure_plugins(vconfig.fetch('vagrant_plugins')) if vconfig.fetch('vagrant_insta
 
 trellis_config = Trellis::Config.new(root_path: ANSIBLE_PATH)
 
-Vagrant.require_version '>= 2.1.0', '<= 2.2.19'
```
Comment (SteelWagstaff) [inline Vagrantfile:19]: @cmurtagh you had previously added an `=` to support Ansible 2.2.19, which caused a merge conflict. I resolved it by accepting the upstream changes seen below

Comment (SteelWagstaff) [reply Vagrantfile:19]: Changes should be safe, since I don't think we're using Darwin in any of our environments :joy:

**Review by cmurtagh — APPROVED** [review]
The config:

`ansible_connection=local` 

Was that added by us?

**Comment by SteelWagstaff** [comment]
No -- it was from the upstream Trellis folks. See https://github.com/pressbooks/trellis/commits/734e262d85b38e656000a8abdac256dd1a88b649/hosts/development for commit history

**Pull request merged**